.github/workflows/dispatch.yml

```diff
@@ -0,0 +1,32 @@
+name: Dispatch workflows
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  dispatch-dc:
+    name: Dispatch DC workflow
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3    
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            src:
+              - 'libs/**'    
+      - if: steps.changes.outputs.src == 'true'
+        name: Trigger DC Workflow
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.DC_PAT }}
+          script: |
+            github.rest.actions.createWorkflowDispatch({
```

Comment:
do you need error handling for this

Reply:
The action library actions/github-script@v6 should have handled most errors. Not sure what is not handled. We will handle it when we see it.